app/controllers/ruleset_controller.rb

```diff
@@ -56,17 +56,15 @@ def create
         @spectral_results = JSON.parse(Linters::Spectral.new(file: ruleset_params[:oas_file]).lint_to_json)
       end
     rescue StandardError => e
-      puts e.message
+      Rails.logger.debug e.message
```

Comment:
Please see my comments on previous PRs. 

A much better solution would be to remove this rescue clause. All that is doing is making debugging harder.

Reply:
Rubocop made this change, I didn't intend on touching this part of the code until the spectral code had been refactored. I think there will be another pr soon to remove this.